rules/common.smk

```diff
@@ -1,2 +1,22 @@
 def is_single_end(sample, unit):
     return pd.isnull(units.loc[(sample, unit), "fq2"])
+    
+def strandedness(units):
```

Comment:
Can you move the strandedness logic into the count-matrix script? Since it is only used there, we can as well hide this detail away, and only make it visible to those who want to know how exactly counts are obtained.

Reply:
No problem